src/components/EnvironmentOverride/DeploymentTemplateOverrideForm.tsx

```diff
@@ -196,12 +199,12 @@ export default function DeploymentTemplateOverrideForm({
         }
     }
     const handleChangeCheckbox = () => {
```

Comment:
it's a good practice to keep a line space between two function definitions

Reply:
done

Reply:
done